source/frontend/src/features/mapSideBar/compensation/detail/CompensationRequisitionDetailView.tsx

```diff
@@ -215,14 +252,16 @@ export const CompensationRequisitionDetailView: React.FunctionComponent<
                   toolTip={cannotEditMessage}
                 />
               )}
-              <StyledAddButton
-                onClick={() => {
-                  onGenerate(compensation);
-                }}
-              >
-                <FaMoneyCheck className="mr-2" />
-                Generate H120
-              </StyledAddButton>
+              {fileType === ApiGen_CodeTypes_FileTypes.Acquisition && (
```

Comment:
hmm, is generation for leases coming later?

Reply:
Yes